pkg/kubectl/cmd/get.go

```diff
@@ -217,7 +218,7 @@ func RunGet(f cmdutil.Factory, out, errOut io.Writer, cmd *cobra.Command, args [
 			return err
 		}
 		if len(infos) != 1 {
-			return fmt.Errorf("watch is only supported on individual resources and resource collections - %d resources were found", len(infos))
+			return fmt.Errorf(i18n.T("watch is only supported on individual resources and resource collections - {{.Count}} resources were found", len(infos)))
```

Comment:
Need to switch to just plain NewError to avoid double-interpreting strings. (This one doesn't matter but others will and people will copy-pasta this.)